src/Components/Notifications/__tests__/Notification.jest.tsx

```diff
@@ -0,0 +1,38 @@
+import { screen } from "@testing-library/react"
+import { setupTestWrapperTL } from "DevTools/setupTestWrapper"
+import { graphql } from "react-relay"
+import { NotificationFragmentContainer } from "Components/Notifications/Notification"
+import { Notification_test_Query } from "__generated__/Notification_test_Query.graphql"
+
+jest.unmock("react-relay")
+
+const { renderWithRelay } = setupTestWrapperTL<Notification_test_Query>({
```

Comment:
fwiw, I made some small improvements to `setupTestWrapperTL` to accept query renderers and therefore not need to define `Component` + `Query` and instead you can define the component using hooks and it should work fine as we do in Eigen.

Reply:
Nice! At the moment the query is defined in [notificationRoutes.ts](https://github.com/artsy/force/blob/542aea7565f54ec1e130e67d80b6b9697d989592/src/Apps/Notifications/notificationsRoutes.ts#L21), so the component doesn't has a query defined. But this might change in the next PR with the split view implementation. I'll can then in this PR improve the test file to not define it's own query.

Reply:
perfect - thanks for explaining @olerichter00. Please feel free to merge this